rasa_core/processor.py

```diff
@@ -337,7 +337,17 @@ def _schedule_reminders(self, events: List[Event],
                                       args=[e, dispatcher],
                                       id=e.name,
                                       replace_existing=True,
-                                      name=str(e.action_name))
+                                      name=str(e.action_name) + "__sender_id:" + tracker.sender_id)
+
+    def _cancel_reminders(self, events: List[Event], tracker) -> None:
```

Comment:
please add `tracker` type

Reply:
I had already changed that in the PR. Do I still have to do something here?

Reply:
similar here: `tracker: DialogueStateTracker`